Assets/unZENity-VR/Scripts/Settings/SettingsManager.cs

```diff
@@ -65,5 +72,28 @@ private void CheckIfGothicIDirectoryExists()
                     $"Please put in the right absolute path to your local installation.");
         }
 
+        /// <summary>
+        /// Import the settings file from streamingAssetPath to persistentDataPath.
+        /// Since the settings file is in streamingAssetPath, we need to use UnityWebRequest to move it so we can have access to it
+        /// as detailed here https://docs.unity3d.com/ScriptReference/Application-streamingAssetsPath.html
+        /// </summary>
+        private void Importer()
+        {
+            string GameSettingsPath = System.IO.Path.Combine(Application.streamingAssetsPath, $"{SETTINGS_FILE_NAME}");
+            string result = "";
+            if (GameSettingsPath.Contains("://") || GameSettingsPath.Contains(":///"))
```

Comment:
I think it will either always be true or always be false. 🤔

Reply:
It's just a check to see if we have a URL, or a path from a jar (ex `"jar:file://`) if not just copy from one place to another